client/src/App.js

```diff
@@ -16,18 +16,27 @@ import AddSeeker from "./pages/AddSeeker";
 import AddEmployee from "./pages/AddEmployee";
 import GetAllEmployees from "./pages/GetAllEmployee";
 import JobList from "./pages/JobList";
+import Cookies from "js-cookie";
 
 function App() {
   const location = useLocation();
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
 
-  // Check if the current path is "/dashboard"
-  const isDashboardPage = location.pathname === "/dashboard";
-
+  useEffect(() => {
+    const userId = Cookies.get("userId");
+    const isLoggedIn = userId !== undefined && userId !== "";
+    setIsLoggedIn(isLoggedIn);
+  }, []);
+  const isDashboardPage =
+    location.pathname.includes("/dashboard") ||
+    (isLoggedIn && location.pathname === "/");
   return (
     <div className="w-screen h-screen bg-white flex flex-col">
       {!isDashboardPage && <NavbarCmp />}
       <Routes>
-        <Route path="/" element={<Home />} />
+        <Route path="/" element={isLoggedIn ? <Dashboard /> : <Home />} />
+        <Route path="/dashboard" element={<Dashboard />} />
```

Comment:
What is diff between /dashboard and /dashboard/employee ?

Reply:
it was earlier needed, have removed it now.